src/routes/console/project-[project]/auth/+page.svelte

```diff
@@ -102,6 +105,21 @@
                                 </Pill>
                             </Copy>
                         </TableCell>
+                        <TableCell onlyDesktop title="Labels">
+                            {#each user.labels.slice(0, 2) as label, i}
+                                {#if i == 0}
+                                    <span
+                                        use:tooltip={{
+                                            disabled: user.labels.length < 2,
+                                            content: user.labels.join(', ')
+                                        }}>
+                                        <Pill>{label}</Pill>
```

Comment:
@TorstenDittmann, if we want to be able to trim the pill, I think the pill code needs to be updated.